### ISSUE TITLE
GOES Clear-sky ABI JEDI YAML file test in MPAS_JEDI 12km case

### ISSUE THREAD

**Issue by xyzemc** [NOAA-EMC/RDASApp#231]
This issue will address the work to add the goes abi **clear-sky** yaml file into RDASApp.

Step1:  **Clear sky** ABI gsrcsr bufr data has been converted to ioda data with python version bufr2ioda issue [#221](https://github.com/NOAA-EMC/RDASApp/issues/221)
Both goes_16 and goes_18 ioda data are ready for testing

Step2: Converted the satbias to  abi_g16.satbias.nc4, and created abi_g16.tlapse.txt

Step3: Copied the  abi_g16 yaml file from GDASApp and modified it for suitable with MPAS model

Step4: The abi_g16 yaml file with all QC filters and bias correction has been successfully tested in MPAS_JEDI

Step5: Validating the results is still ongoing

**Comment by xyzemc** [comment]
Data count from each QC filter category:
![image](https://github.com/user-attachments/assets/ae87e66d-a824-4bf7-810b-1ff2a5f7ca4d)

Total data count for each channel passed the QCs:
![image](https://github.com/user-attachments/assets/3e71fcfc-2f33-449f-a617-4032a8b71aaa)

**Comment by xyzemc** [comment]
Before QC: Hofx Brightness Temperature from GOES_16 ABI
![image](https://github.com/user-attachments/assets/070ffb1e-8017-45d0-a60d-aa162b663b6f)
![image](https://github.com/user-attachments/assets/4a4b0662-afdf-4d30-b9ab-1303969e6546)
![image](https://github.com/user-attachments/assets/0735702c-546e-414c-9629-4919b5febffe)

After QC filtered: Hofx Brightness Temperature
Thinning to 145km
![image](https://github.com/user-attachments/assets/fe72c7d2-fcce-4828-a386-2851392ac54f)
![image](https://github.com/user-attachments/assets/6a412816-c9be-40a0-9464-83ac50c37822)
![image](https://github.com/user-attachments/assets/8fcf45ae-ff18-492f-bc2e-c436098f4e8e)

Thinning to 60km:
![image](https://github.com/user-attachments/assets/99f3652d-318e-4025-939d-2c5b8f2a9fd8)
![image](https://github.com/user-attachments/assets/66dc6b78-ac46-444d-8aa6-6001969534fb)
![image](https://github.com/user-attachments/assets/ff868f99-e47c-4f2c-ada8-20b4cd80cb55)

**Comment by xyzemc** [comment]
**GOES_16 ABI OmB with QC & BC**
![image](https://github.com/user-attachments/assets/8ed96676-644e-4f3a-8b1c-638ca41c49c9)
![image](https://github.com/user-attachments/assets/5366d6c6-ee36-4f12-ae8c-6191885885d2)
![image](https://github.com/user-attachments/assets/34151ac1-4aa9-458f-9f19-86e6c4643eef)


**Total Bias**
![image](https://github.com/user-attachments/assets/9d019acc-d6f8-4c5b-b68c-a7c1314f0c98)
![image](https://github.com/user-attachments/assets/523752d7-ce2d-4704-a15a-3768e6c54666)
![image](https://github.com/user-attachments/assets/e4ca1ab2-7996-4fe7-825b-960a5a57c603)

**Comment by xyzemc** [comment]
ABI Weighting Function:
![image](https://github.com/user-attachments/assets/a66f6a92-5d12-44f7-bd5f-f20c6d68835e)

**Comment by xyzemc** [comment]
Since JEDI assimilated three water vapor channels (8, 9, 10) peaking between 600-300 hPa, we will check the Increments at level 23-36:
Here are the increment from only assimilate **GOES_16 ABI** radiance data:
![image](https://github.com/user-attachments/assets/30c91bca-6434-4c03-8b66-46eeabbbf32e)
![image](https://github.com/user-attachments/assets/727636a8-f41e-44f6-b8de-1c75c8991d94)
![image](https://github.com/user-attachments/assets/221051a1-df16-4c2a-8c74-69ea8ae9bf59)

**Comment by guoqing-noaa** [comment]
@xyzemc Does this include all-sky radiance data? Thanks!

**Comment by xyzemc** [comment]
> @xyzemc Does this include all-sky radiance data? Thanks!

No, this only clear sky data. I will update the issue title to make it clear.

**Comment by guoqing-noaa** [comment]
> > @xyzemc Does this include all-sky radiance data? Thanks!
> 
> No, this only clear sky data. I will update the issue title to make it clear.

Thanks, @xyzemc

**Comment by xyzemc** [comment]
**GOES_18 ABI** radiance data has been tested successfully.
Before QC: Hofx Brightness Temperature from **GOES_18 ABI**
![image](https://github.com/user-attachments/assets/61922e84-bb60-4c41-ae35-6dfa23247c14)
![image](https://github.com/user-attachments/assets/7a1650ca-9d26-40f8-96c9-c35901d32fe9)
![image](https://github.com/user-attachments/assets/8fe093fc-100f-4d7d-8ea8-c1d896442b41)

After QC:
![image](https://github.com/user-attachments/assets/204ed81b-9bd8-4679-91e5-9ddf9fdd558e)
![image](https://github.com/user-attachments/assets/9e378418-539b-4ca3-80e5-1eae406d0e4f)
![image](https://github.com/user-attachments/assets/f0eb78fb-3fc8-4708-ab0d-2e99a654db73)

**Comment by xyzemc** [comment]
**GOES_18 ABI OmB with QC & BC**
![image](https://github.com/user-attachments/assets/ab914da3-b6b7-4004-b892-37c425bff8eb)
![image](https://github.com/user-attachments/assets/4d6e72e5-dc5c-479c-9eb5-6064578936e8)
![image](https://github.com/user-attachments/assets/04bb71e7-3766-415f-a926-da771c51e72a)

**Comment by xyzemc** [comment]
**Analysis increment with assimilation of GOES_18 ABI water vapor channels:**
![image](https://github.com/user-attachments/assets/0092d50a-5882-4b9e-a43f-88ac0c1115cd)
![image](https://github.com/user-attachments/assets/da09cffd-abfb-4251-b0db-f905d8d11000)
![image](https://github.com/user-attachments/assets/22f50f85-bd43-45f3-9a95-9bf71ba4c3db)
